### ISSUE TITLE
Issue when using next()

### ISSUE THREAD

**Issue by ed5555** [kernc/backtesting.py#1184]
When trying to run a simple 200 MA strategy, I want make a purchase at the onset of the data. However, the next function iterates only from the onset of the data+200 days.

Is there a way to do this?

Thank you

**Comment by sachaRfd** [comment]
When implementing a strategy with 200 MA, you need to account for the initial calculation period. For example with a 200-day MA, you need the data of the previous 200 days before you can generate a valid MA value. During the calculation period, the MA will be NaN - so no signal.  This means the strategy will only work from 200 days onwards. 

Quick fixes in your case would be to either: 

1. Collect more historical data to your desired start-date.
2. Pick a later start date where you already have the previous 200 data points.

Hope that makes sense.

**Comment by kernc** [comment]
Indeed, mind the warning on [`Backtest.run`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Backtest.run).

Welcome a better position where this note should be placed.
